### PR TITLE
Build the funding info page

### PR DESCRIPTION
**Pull request by tavy88**
### Context

- new static page built to give info about funding options
- outbound links tracked and track who visits the new page
- can be reached from action plan
- add hyperlink to action plan at bottom of “Your training choices” section
- update content for action plan if nothing is selected, to mention that courses are often free etc.

### Ticket
https://dfedigital.atlassian.net/browse/GET-829

### CODE REVIEW

File: app/views/pages/funding_info.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<% page_title :pages_funding_info %>
```
Comment (cwrw) [inline app/views/pages/funding_info.html.erb:1]: can you add `pages_funding_info` to the translations file under `page_titles`?

File: app/views/pages/funding_info.html.erb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <h2 class="govuk-heading-m">Free training</h2>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>Many people can study maths and English for free.</li>
+      <li>You can study computer skills online for free on the <%= link_to 'Learn my Way', 'https://www.learnmyway.com/subjects', class: 'govuk-link' %> website.</li>
+    </ul>
+    <p class="govuk-body">Other free courses are available. Speak to a local adviser to find out what’s available in your area. Call free on <%= link_to t('contact_us.telephone'), t('contact_us.telephone_link'), class: 'govuk-link' %> Monday to Friday from 9am to 5pm.</p>
+    <h2 class="govuk-heading-m">Help with training costs</h2>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>You may be able to get funding to do courses that you need to pay for. This includes grants and bursaries, which you may not need to pay back.</li>
+        <li>If you’re facing financial hardship you could get extra help, including funding for your travel and accommodation costs, course materials and equipment, and childcare costs.</li>
```
Comment (cwrw) [inline app/views/pages/funding_info.html.erb:20]: could you fix indentation here?

File: config/routes.rb [hunk 3]
```diff
@@ -36,6 +36,7 @@
     get 'cv-advice', to: 'pages#cv_advice'
     get 'cover-letter-advice', to: 'pages#cover_letter_advice'
     get 'interview-advice', to: 'pages#interview_advice'
+    get 'funding-info', to: 'pages#funding_info'
```
Comment (cwrw) [inline config/routes.rb:39]: I've enquired about this and we should use ` funding-information` instead as it's not an abbreviation

File: app/views/pages/training/_with_options.html.erb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      <div class="govuk-grid-column-one-half">
+        <h3 class="govuk-heading-s">Computer skills training</h3>
+      </div>
+      <div class="govuk-grid-column-one-half">
+        <%= link_to 'Go to learning provider website', 'https://www.learnmyway.com/subjects', class: 'govuk-button govuk-button--secondary action-plan-action', data: { module: 'govuk-button' } %>
+      </div>
+    </div>
+  <% end %>
+</div>
+<p class="govuk-body govuk-!-margin-top-3">You can find other types of training by searching on Find a course, run by the National Careers Service.</p>
```
Comment (cwrw) [inline app/views/pages/training/_with_options.html.erb:38]: I think this content should be similar to the without options version. We intentionally added `this button will take you to another government service.` I think this has been missed from the prototype by accident

File: config/routes.rb (hunk 3 above)
Comment (tavy88) [reply config/routes.rb:39]: Ok.

File: app/views/pages/training/_with_options.html.erb (hunk 4 above)
Comment (tavy88) [reply app/views/pages/training/_with_options.html.erb:38]: Needs checking with Lois.

**Pull request merged**